src/server.h

```diff
@@ -802,6 +802,7 @@ typedef struct client {
     size_t querybuf_peak;   /* Recent (100ms or more) peak of querybuf size. */
     int argc;               /* Num of arguments of current command. */
     robj **argv;            /* Arguments of current command. */
+    size_t argv_sum;        /* Sum of lengths of objects in argv list. */
```

Comment:
I'd consider a more descriptive name, e.g. `argv_mem`.

Reply:
i wanted to denote that it's the sum of lengths (chars) and not memory (includes internal frag and sds unused space).
maybe you have a better suggestion?

Reply:
argv_lens?

Reply:
used `argv_len_sum`